backend/clubs/views.py

```diff
@@ -2671,9 +2692,9 @@ def create_tickets(self, request, *args, **kwargs):
             Ticket(
                 event=event,
                 type=item["type"],
-                price=item["price"],
-                group_discount=item.get("group_discount", None),
-                group_size=item.get("group_size", None),
+                price=item.get("price", 0),
+                group_discount=item.get("group_discount", 0),
+                group_size=item.get("group_size", 0),
```

Comment:
Note: setting this to 0 won't change how cart totals are calculated, since we check `if group_size` before applying the group discount in `initiate_checkout`. However, it's a little unclear, and may trip us up later. What do you think of either 1) defaulting `group_size` to 0 in the `Ticket` model, and checking `if group_size != 0` or 2) reverting this line to `item.get("group_size", None)`?

Reply:
Will revert. Looks like we only need to non-nullify `group_discount`?

Reply:
Yup, since it defaults to 0 in the model definition.

Reply:
Also just noticed that you've defined `group_size` as a PositiveIntegerField so `group_size` = 0 would have also failed at runtime!